client/src/App.test.js

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { FetchMock } from '@react-mock/fetch';
+import { render, wait } from '@testing-library/react';
 import App from './App';
 
-test('loads and displays text from Top Charts', () => {
-    const { getByText } = render(<App />);
-    const textElement = getByText('Top Charts');
-    expect(textElement).toBeInTheDocument();
+const renderComponent = () => 
+  render(
+      <FetchMock
+        options={{
+          matcher: '/api/v1/stream', method: 'GET', response: {"clusters":[{"title":"Top Charts","subtitle":"For Wear OS","charts":[{"title":"Top Free","apps":[{"id":"1","icon":"mockIcon1.png","title":"mockApp1","category":"Fake Apps","rating":5.0,"price":0.0},{"id":"2","icon":"mockIcon2.png","title":"mockApp2","category":"Fake Apps","rating":5.0,"price":0.0}]},{"title":"Top Grossing","apps":[{"id":"3","icon":"mockIcon3.png","title":"mockApp3","category":"Fake Apps","rating":5.0,"price":0.0},{"id":"4","icon":"mockIcon4.png","title":"mockApp4","category":"Fake Apps","rating":5.0,"price":0.0}]},{"title":"Top Paid","apps":[{"id":"5","icon":"mockIcon5.png","title":"mockApp5","category":"Fake Apps","rating":5.0,"price":0.0},{"id":"6","icon":"mockIcon6.png","title":"mockApp6","category":"Fake Apps","rating":5.0,"price":0.0}]}],"type":"TopCharts"},{"cards":[{"title":"mockCard1","bigImageUrl":"https://mockBigImageUrl.com","apps":{"id":"1","icon":"mokcIcon1.png","title":"mockApp1","category":"Fake Apps","rated":"Everyone","appDescription":"mock app description","rating":5.0,"price":0.0}},{"title":"mockCard2","bigImageUrl":"https://mockBigImageUrl.com","apps":{"id":"2","icon":"mokcIcon2.png","title":"mockApp2","category":"Fake Apps","rated":"Everyone","appDescription":"mock app description","rating":5.0,"price":0.0}},{"title":"mockCard3","bigImageUrl":"https://mockBigImageUrl.com","apps":{"id":"3","icon":"mokcIcon3.png","title":"mockApp3","category":"Fake Apps","rated":"Everyone","appDescription":"mock app description","rating":5.0,"price":0.0}},{"title":"mockCard4","bigImageUrl":"https://mockBigImageUrl.com","apps":{"id":"4","icon":"mokcIcon4.png","title":"mockApp4","category":"Fake Apps","rated":"Everyone","appDescription":"mock app description","rating":5.0,"price":0.0}}]}]}
```

Comment:
Can you make a variable for the response and move the data into the var so this is cleaner to read?

```
const responseData = {"clusters": ...};

...
matcher: '/api/v1/stream', method: 'GET', response: responseData

```